src/site/content/en/handbook/web-dev-components/index.md

```diff
@@ -1095,5 +1225,135 @@ assumenda perspiciatis.
   </table>
 </div>
 
+## Tabs
+Tabs are currently only designed for use in callouts.
+Don't use tabs in the main post body because there's no visual indicator
+of where tab content stops and main post content resumes.
+
+You can use three keywords in the Tab shortcode argument
+to generate sequentially numbered tab labels:
+- `question`: creates the label `Question n`, where _n_ is the number of the tab in the set.
+- `sample`: creates the label `Sample n`.
+- `bare`: creates the label `n`.
+
+Any other text in the Tab shortcode argument will be presented as-is.
+
+````html
+{% raw %}{% Tabs 'Questions for knowledge self check' %}
+{% Tab 'question' %}
+
+Lorem ipsum [dolor sit amet](#), consectetur adipiscing elit. Proin dictum a
+massa sit amet ullamcorper. `Suspendisse` auctor ultrices ante, nec tempus
+nibh varius at.
+
+{% endTab %}
+{% Tab 'question' %}
+
+```html
+<label for="inky">
+  Inky
+  <input type="radio" id="inky" name="ghosts" value="inky">
+</label>
+```
+
+{% endTab %}
+{% Tab 'Custom' %}
+
+Lorem ipsum [dolor sit amet](#), consectetur adipiscing elit.
```

Comment:
There seem to be a styling issue in tabs here:
![Screenshot 2019-12-17 at 4 10 17 PM](https://user-images.githubusercontent.com/1914261/71008148-72dfea00-20e8-11ea-96d3-1c1d9b15e8a1.png)

Reply:
Indeed! That's an issue with the `web-copy-code` component, not the components in this PR. I've created a ticket (#2010).